cinema/serializers.py

```diff
@@ -1 +1,99 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    CinemaHall,
+    Genre,
+    Actor,
+    Movie,
+    MovieSession,
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
```

Comment:
The `capacity` field is included in the `fields` tuple. Ensure that the `CinemaHall` model has a `capacity` field, or remove it from the serializer if it does not exist.